docker-compose.yml

```diff
@@ -86,6 +86,14 @@ services:
     environment:
       - POSTGRES_HOST=postgres_docker
       - NODE_ENV=development
+  sftp:
+    image: atmoz/sftp
+    container_name: monitoring-sftp
+    volumes:
+        - ./test-sftp:/home/tta_ro/ProdTTAHome
+    ports:
+        - "22:22"
```

Comment:
22 was tied up by default on my mac, looks like it might be something internal, anyway we could map it to something else?

```bash
 netstat -na | grep "22"     

tcp4       0      0  *.22                   *.*                    LISTEN     
tcp6       0      0  *.22                   *.*                    LISTEN

Reply:
Seconding the suggestion to use .env to determine port here. Having the same issues as @thewatermethod running this locally.

Reply:
I have plumbed the ENV defining the port for the FTP server to also configure the ftp docker container. This means that it now can run on any port.